wiki/content/deploy/dgraph-administration.md

```diff
@@ -6,7 +6,41 @@ weight = 18
     parent = "deploy"
 +++
 
-Each Dgraph Alpha exposes administrative operations over HTTP to export data and to perform a clean shutdown.
+Each Dgraph Alpha exposes various administrative (admin) endpoints both over
+HTTP and GraphQL, for example endpoints to export data and to perform a clean
+shutdown. All such admin endpoints are protected by three layers of authentication:
+
+1. IP White-listing (use `--whitelist` flag in alpha to whitelist IPs other than
+   localhost).
+2. Poor-man's auth, if alpha is started with the `--auth_token` flag (means you
+   will need to pass the `auth_token` as `X-Dgraph-AuthToken` header while
+   making the HTTP request).
+3. Guardian-only access, if ACL is enabled (means you need to pass the ACL-JWT
+   of a Guardian user using the `X-Dgraph-AccessToken` header while making the
+  HTTP request).
+
+An admin endpoint is any HTTP endpoint which provides admin functionality.
+Admin endpoints usually start with the `/admin` path. The current list of admin
+endpoints includes the following:
+
+* /admin
+* /admin/backup
+* /admin/config/lru_mb
+* /admin/draining
+* /admin/export
+* /admin/shutdown
+* /admin/schema
+* /alter
+* /login
```

Comment:
Just for consistency, I'd suggest using code formatting for the endpoint list too:

```
* `/admin`
* `/admin/backup`
...
* `/login`
```